cf-231A.mdx

```diff
@@ -0,0 +1,154 @@
+---
+id: cf-231A
+source: CF
+title: Team
+author: Sathvik Chundru
+---
+
+[Official Analysis (CPP)](https://codeforces.com/blog/entry/5486)
+
+## Solution (Greedy)
+
+Let's consider the following test case:
+
+```
+3
+1 1 0
+1 1 1
+1 0 0
+```
+
+There are 2 ways to solve this problem. One solution is to input all the ones
+and zeros into a two-d array and sort through them manually to see how many
+cases have two or more ones. The second simpler solution is to input three
+integers per row and check if their sum is greater than or equal to two. Both
```

Comment:
Don't these two things mean the same?

Reply:
? they are 2 different approaches but that's besides the point sans wants me to remove the worse of the 2 solutions so I might do that someone else from team/core tell me to and ill do it

Reply:
I think the idea is the same, just different implementation so I agree that you should just keep the second solution.